otelib/settings.py

```diff
@@ -10,6 +12,10 @@ class Settings(BaseModel):
         (3.0, 27.0), description="Tuple for URL connect and read timeouts in seconds."
     )
 
+    host: str = Field(
+        DEFAULT_HOST, description="Host with oteapi-services up and running."
+    )
```

Comment:
I'd probably call this something `services`-related to pin it semantically to the `services` backend. So maybe `services_host` or similar? And then perhaps also make the default a fully resolvable URL, i.e., `http://localhost:8080` or similar.

Hmm, thinking more about this. It might be nice to match this up with a system environment variable for developers that matches a running OTEAPI Services server... But at the same time, I'd like to make it clear that this is a backend-specific configuration option. Hmm. Tricky.

Reply:
I just renamed it into `remote_backend` and added some description. However, it is not as default when the `client` is initialized for the serivces, which pretty much makes this attribute useless in the settings.

Shall we also remove it?

Reply:
Done.